Pipeline/pipeline.py

```diff
@@ -41,11 +58,11 @@ def HTML_formater(df, name):
                    </body>
                     </html>.
                """
-    with open("tables/{}.html".format(name), "w") as f:
+    with open(path / "tables/{}.html".format(name), "w") as f:
         f.write(html_string.format(table=df.to_html(classes="mystyle")))
 
 
-class Pipeline:
+class Pipeline(object):

```

Comment:
In python 3 we do not need to inherit from object.

Reply:
Fixed